crypto/http/http_client.c

```diff
@@ -377,10 +376,10 @@ static int parse_http_line1(char *line, int *found_keep_alive)
     int i, retcode;
     char *code, *reason, *end;
 
-    if (strncmp(line, HTTP_PREFIX_VERSION, HTTP_VERSION_PATT_LEN) != 0)
+    if (strncmp(line, HTTP_PREFIX_VERSION, strlen(HTTP_PREFIX_VERSION)) != 0)
```

Comment:
Why not `sizeof(HTTP_PREFIX_VERSION) - 1`?

Reply:
Did not think of that.
Well, the advantage is the static evaluation, but I find the needed extra `- 1` risky/confusing.

Reply:
After thinking more about this, I replaced all the uses of `strlen` on constant strings.